scripts/release/createWebModules.js

```diff
@@ -160,23 +146,30 @@ async function createChartsModule() {
         isDraft: true
     });
     await execShellCommand(`rm -rf ${tmpFolder}`);
-    console.log("Done.");
-}
+
     console.log("Done.");
 }
 
-async function commonActions(moduleInfo, widgets = []) {
+async function commonActions(moduleInfo, widgets = [], exportWithoutSnippet = false, updateWidgetsChangelogs = false) {
     const tmpFolder = join(repoRootPath, "tmp", moduleFolderNameInRepo);
     await githubAuthentication(moduleInfo);
-    const moduleChangelogs = await updateModuleChangelogs(moduleInfo);
+    const moduleChangelogs = await (updateWidgetsChangelogs ? updateChangelogs : updateModuleChangelogs)(moduleInfo);
     if (!moduleChangelogs) {
         throw new Error(
             `No unreleased changes found in the CHANGELOG.md for ${moduleInfo.nameWithSpace} ${moduleInfo.version}.`
         );
     }
     await commitAndCreatePullRequest(moduleInfo);
-    await updateTestProjectWithWidgetsAndAtlas(moduleInfo, tmpFolder, widgets);
+    if (exportWithoutSnippet) {
```

Comment:
What snippet?

Reply:
In the module if there is no snippet widgets are not exported, so this flag is just to identify if the project constains snippets or not.